ons_alpha/release_calendar/models.py

```diff
@@ -17,16 +17,17 @@
 
 
 class ReleaseStatus(models.TextChoices):
-    UPCOMING = "UPCOMING", "Upcoming"
-    PUBLISHED = "PUBLISHED", "Published"
-    CANCELLED = "CANCELLED", "Cancelled"
+    PROVISIONAL = "provisional", "Provisional"
```

Comment:
`ReleasePage.get_template()` is still checking for `UPCOMING`

Reply:
Question: Should these strings be translated? See the example in Django's docs: https://docs.djangoproject.com/en/5.1/ref/models/fields/#enumeration-types